scripts/bundle.js

```diff
@@ -53,3 +56,11 @@ function bundleOne(moduleId, exclude) {
 		fs.writeFileSync(filePath, BUNDLED_FILE_HEADER + result.code);
 	})
 }
+
+function updateImports(moduleId) {
+	console.log(`ESM: updating relative imports paths for ${moduleId}...`);
```

Comment:
Remove this log statement?  Seems like it could be useful during debugging but not long term.

```suggestion
```